Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -150,19 +151,7 @@ void UPSWorldSubsystem::OnWorldBeginPlay(UWorld& InWorld)
 
 		if (GameFeatureName == "ProgressionSystem")
```

Comment:
- Comparing with `ProgressionSystem` does not make sense from `OnWorldBeginPlay`. It's already called when your world subsystem is loaded. Please remove that unnecessary code
- Once you remove all that code above, I don't see any point in `WolrdSubSystemInitialize` function. Just put all the logic here then

Reply:
removed: https://github.com/h4rdmol/ProgressionSystem/pull/26/commits/927dc6b9f95aa8289ea9acc342dd3d3f2560f391